Sources/Plasma/PubUtilLib/plStatGather/plAutoProfile.cpp

```diff
@@ -39,40 +39,41 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
       Mead, WA   99021
 
 *==LICENSE==*/
+
 #include "plAutoProfile.h"
+#include "plProfileManagerFull.h"
 
 #include "plgDispatch.h"
-#include "plNetClient/plNetClientMgr.h"
-#include "plNetClient/plNetLinkingMgr.h"
-
+#include "hsResMgr.h"
 #include "hsStream.h"
 #include "hsTimer.h"
-#include "plMessage/plAgeLoadedMsg.h"
 #include "plTimerCallbackManager.h"
-#include "plMessage/plTimerCallbackMsg.h"
-#include "plAvatar/plAvatarMgr.h"
-#include "plAvatar/plArmatureMod.h"
-#include "plModifier/plSpawnModifier.h"
-#include "plMessage/plConsoleMsg.h"
-#include "pnMessage/plClientMsg.h"
-#include "plAgeLoader/plAgeLoader.h"
-#include "plProfileManagerFull.h"
+#include "hsWindows.h"
 
-#include "plPipeline/plDebugText.h"
+#include "pnMessage/plClientMsg.h"
 #include "pnMessage/plTimeMsg.h"
 
+#include "plAvatar/plArmatureMod.h"
+#include "plAvatar/plAvatarMgr.h"
+#include "plContainer/plConfigInfo.h" // for plStringList
+#include "plMessage/plAgeLoadedMsg.h"
+#include "plMessage/plConsoleMsg.h"
+#include "plMessage/plTimerCallbackMsg.h"
+#include "plModifier/plSpawnModifier.h"
+#include "plNetClient/plNetClientMgr.h"
+#include "plNetClient/plNetLinkingMgr.h"
+#include "plPipeline/plDebugText.h"
 #include "plStatusLog/plStatusLog.h"
 #include "plVault/plVault.h"
 
-#include "plContainer/plConfigInfo.h" // for plStringList
-
 // For taking screenshots
-#include "plGImage/plMipmap.h"
-#include "../../Apps/plClient/plClient.h"
-#include "plGImage/plJPEG.h"
 #include "plPipeline.h"
+#include "../../Apps/plClient/plClient.h" // FIXME FIXME FIXME DAMMIT
```

Comment:
:(

Reply:
Ugh. Yeah. IIRC this was already present and way outside the scope of what I was trying to do. Ideally, we'd convert all that stuff to use `plClientMsg` or something like that.